### PR TITLE
use code-demo tag

### PR DESCRIPTION
**Pull request by randcode-generator**
https://github.com/codelab-fun/codelab/issues/1022

### CODE REVIEW

**Comment by randcode-generator** [comment]
Screen Shot 2019-09-06 at 2 10 56 PM
Screen Shot 2019-09-06 at 2 12 05 PM

File: apps/codelab/src/app/codelabs/angular/templates/samples/data-binding-extra/app.component.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<h1>Here we bind value of the input to the
+    name property </h1>
+  <input [value]="name">
```
Comment (NothingEverHappens) [inline apps/codelab/src/app/codelabs/angular/templates/samples/data-binding-extra/app.component.html]: This seems weirdly formatted

File: apps/codelab/src/app/codelabs/angular/templates/samples/data-binding-extra/app.component.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  
+  <h1>A shortcut applying class (or not) based
+    on the value of isSpecial</h1>
+  <div [class.special]="isSpecial">
+    Wow, I'm special!
+  </div>
+  
+  <h1>Shortcut for binding styles</h1>
+  <button [style.color]="isSpecial ? 'gree' : 'gray'">
```
Comment (NothingEverHappens) [inline apps/codelab/src/app/codelabs/angular/templates/samples/data-binding-extra/app.component.html]: green

File: apps/codelab/src/app/codelabs/angular/templates/samples/data-binding-extra/app.component.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    special button
+  </button>
+  
+  <h1>It also works with custom components</h1>
+  <birthday-card [name]="name"></birthday-card>
```
Comment (NothingEverHappens) [inline apps/codelab/src/app/codelabs/angular/templates/samples/data-binding-extra/app.component.html]: add a line

File: apps/codelab/src/app/codelabs/angular/templates/samples/data-binding-extra/app.component.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'my-app',
+  templateUrl: './app.component.html',
+})
+export class AppComponent {
+  readonly name = 'Camille Pissarro';
+  isSpecial = true;
+}
```
Comment (NothingEverHappens) [inline apps/codelab/src/app/codelabs/angular/templates/samples/data-binding-extra/app.component.ts]: add new line

**Pull request merged**